### PR TITLE
Draw add command Activity Diagram 

### PR DESCRIPTION
**Pull request by DennieDan**
There are 2 proposed diagrams
* Straight forward, and
* Swimlane version

__Note:__
- Swimlane version is quite small, trying to find ways to improve
- But can choose either of them

image
image

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -158,7 +158,16 @@ Classes used by multiple components are in the `seedu.address.commons` package.
 
 This section describes some noteworthy details on how certain features are implemented.
 
-### \[Proposed\] Undo/redo feature
+### Add a new student
+On execution, the new created `Student` is then be checked if it is a duplicate of any `Student` in the current `AddressBook`, i.e. it has the exactly same `Name` **and** `Phone` with any of the current `Student`s. A `CommandException` will be thrown if it is a duplicate.
```
Comment (Soumyadip-cmd) [inline docs/DeveloperGuide.md:162]: Could consider using this as a template:
image

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -158,7 +158,16 @@ Classes used by multiple components are in the `seedu.address.commons` package.
...
+
+Else, the `Schedule` of the new `Student` is checked if it clashes with any current `Schedule`s so that a warning message will be sent together with the successful message afterward.
```
Comment (Soumyadip-cmd) [inline docs/DeveloperGuide.md:164]: Refer to above comment.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -158,7 +158,16 @@ Classes used by multiple components are in the `seedu.address.commons` package.
...
+
+Below is an activity diagram when [Adding a new student](#add-a-new-student)
+
+<puml src="diagrams/AddCommandActivityDiagram.puml" alt="AddCommandActivityDiagram"/>
+
+<puml src="diagrams/AddActivityDiagram.puml" alt="AddActivityDiagram"/>
```
Comment (Soumyadip-cmd) [inline docs/DeveloperGuide.md:170]: I feel the one without the swim lanes is better.

**Review by Soumyadip-cmd — COMMENTED** [review]
The explanation of the execution part I think would be better to follow the template. I think the first activity diagram looks better.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (DennieDan) [reply docs/DeveloperGuide.md]: Thank you so much for your suggestion. I have done fixing this!

**Review by Soumyadip-cmd — APPROVED** [review]
LGTM

**Pull request merged**